src/FINDOLOGIC/Export/CSV/CSVItem.php

```diff
@@ -127,7 +131,7 @@ private function buildImages(): string
         return $imageUrl;
     }
 
-    private function sanitize($input, $stripTags = true): string
+    private static function sanitize($input, $stripTags = true): string
```

Comment:
Why using a static method here? What advantage does that bring?

Reply:
The question is rather, what would be the advantage of not using a static method? But more profane than that, the IDE was nagging me that this method was needlessly static.